authbridge/demos/github-issue/setup_keycloak.py

```
@@ -1285,18 +1272,113 @@ def _format_user_summary(
+            role_name = mapping["role"]
+
+            if client_name not in client_ids:
+                print(f"  Warning: Client '{client_name}' not found")
+                continue
+
+            client_id = client_ids[client_name]
+
+            try:
+                _add_client_role_to_realm_role_composite(admin, realm, user_role, client_id, role_name)
+                print(f"  ✓ Added client role '{client_name}.{role_name}' to realm role '{user_role}'")
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)